sorc/fvcom_tools.fd/module_ncio.f90

```diff
@@ -2513,13 +2513,14 @@ end subroutine convert_theta2t_2dgrid
   !! @param[in] units units to use in netcdf variable
```

Comment:
When compiling with the doxygen (-DENABLE_DOCS=ON), I got a warning about 'dtype' being undefined. Define it after the definition for 'units'.

Reply:
done

Reply:
Thanks.